gradle.properties

```diff
@@ -1,14 +1,12 @@
-# Project-wide Gradle settings.
+gradle# Project-wide Gradle settings.
```

Comment:
Is `gradle` a property? Should the "Project-wide Gradle settings." string be a on a separate line?

Reply:
That is my mistake. I fixed in the next version